api_docs/group-setting-values.md

```diff
@@ -88,3 +88,29 @@ accidentally reverts the first one without either user noticing.
 Omitting `old` is appropriate where the intent really is a new complete
 list rather than an edit, for example in an integration that syncs the
 list from an external source of truth.
+
+## Permitted values
+
+Not every possible collection of groups is a valid configuration for a
+given group-settings value. For example, as a security hardening
```

Comment:
nit: 

```suggestion
Not every possible group-setting value is a valid configuration for a
given group-based setting. For example, as a security hardening
```

This way we're consistent that a group-settings value is a value, not a receptacle that takes a value. The receptacles that take this type of value are settings.